src/version.h

```diff
@@ -39,4 +39,7 @@ static const int SENDHEADERS_VERSION = 70012;
 //! "feefilter" tells peers to filter invs to you by fee starts with this version
 static const int FEEFILTER_VERSION = 70013;
 
+//! Version after which witness support potentially exists
+static const int WITNESS_VERSION = 70013;
```

Comment:
This variable doesn't appear to be used by anything; I assume this was introduced before switching to the service bit, not sure if you intended to leave it here as a useful marker, or remove?